### PR TITLE
Allow UpdateAdminProfile to run against alternate profiles

### PR DESCRIPTION
**Pull request by davidmreed**
# Critical Changes

# Changes

- `UpdateAdminProfile` now accepts a `profile_name` argument, which defaults to `Admin`. This allows the task to be executed against other profiles.

# Issues Closed

### CODE REVIEW

**Comment by force2b** [comment]
This is awesome!!

**Comment by davidmreed** [comment]
Whoops, fixing the issue with the release test.

File: cumulusci/files/admin_profile.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     </types>
     <types>
         <members>Admin</members>
-        <name>Profile</name>
+        <name>{profile_name}</name>
```
Comment (jstvz) [inline cumulusci/files/admin_profile.xml]: Should you be replacing `<members>` instead?

**Comment by davidmreed** [comment]
> I'd suggest renaming the class from UpdateAdminProfile to UpdateProfile (but also add `UpdateAdminProfile = UpdateProfile` to the module as an alias for backwards compatibility)

There's at least one project with a custom task that imports `UpdateAdminProfile` via the "wrong" path (`from cumulusci.tasks.salesforce.UpdateAdminProfile import UpdateAdminProfile` instead of `from cumulusci.tasks.salesforce ...`), such that if I rename the class and its containing file it will be a breaking change. Is there a preferred pattern for doing that in a backwards-compatible way? Just don't change the filename?

I will fix that project, but it may not be the only one doing such a thing.

**Comment by davisagli** [comment]
@davidmreed oh hmm -- really I'd like to rename all the camel case modules to snake case, but I didn't realize that it was possible to do the import that way.

It should be possible to set up module aliases something like this (within `cumulusci/tasks/salesforce/__init__.py`):

    from cumulusci.tasks.salesforce import update_profile
    UpdateProfile = UpdateAdminProfile = update_profile.UpdateProfile
    sys.modules["cumulusci.tasks.salesforce.UpdateAdminProfile"] = update_profile

**Comment by davidmreed** [comment]
Had no idea I could synthesize the module like that. Thanks, @davisagli. I took the opportunity to snake-case the module as well.

**Pull request merged**